controllers/operator/nmstate_controller.go

```diff
@@ -44,6 +44,7 @@ import (
 	nmstatev1 "github.com/nmstate/kubernetes-nmstate/api/v1"
 	"github.com/nmstate/kubernetes-nmstate/pkg/cluster"
 	nmstaterenderer "github.com/nmstate/kubernetes-nmstate/pkg/render"
+	operatorv1 "github.com/openshift/api/operator/v1"
```

Comment:
The context of this code is not `openshift`
```suggestion
	openshiftoperatorv1 "github.com/openshift/api/operator/v1"
```

Reply:
also changed: `consolev1` --> `openshiftconsolev1`